challenge.rb

```diff
@@ -45,7 +57,17 @@ def okay_to_drink? age, drinking_age
 #   "Fix the AC whenever you have the chance. It's cool."
 
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if ac_working = true && current_temp > desired_temp
+    return "Turn on the AC."
+
```

Comment:
In Ruby, any time you have an `if... elsif... else`, you need an `end`. Anytime you define a method, you need an `end`.